README.md

```diff
@@ -69,5 +68,25 @@ service nginx reload
 - activate the virtualenv (`source bin/activate`)
 - run the instructions in the `deploy.sh` file
 - go to `/etc/systemd/system`
-- create a new service with a symlink 
-- reload the daemon 
+- create a new service with a symlink
+- reload the daemon
+
+## Passwords
+
+Access to shared services (OVH, Twitter, etc.) must be done through individual accounts, to which the management is delegated.
+
+However, some systems do not support delegation, or it can sometimes be helpful to get administrator access to master accounts.
+
+A database containing all the necessary passwords is therefore accessible.
+
+### Instructions
+
+* Install [KeePassX](https://www.keepassx.org)
+* Download the [database](openfisca.kdbx)
+* Request key and password on the [#of-ops](https://openfisca.slack.com) channel.
```

Comment:
So, the targeted user must have access to this slack. Explain earlier who is this documentation for?
> OpenFisca France community administrators? 🤔

Reply:
😢